app/views/unattended/provisioning_templates/snippet/container_certs_setup.erb

```diff
@@ -0,0 +1,51 @@
+<%#
+kind: snippet
+name: container_certs_setup
+model: ProvisioningTemplate
+snippet: true
+description: |
+  This snippet sets up SSL certificates for secure communication with the Foreman registry.
+-%>
+container_certs_setup() {
```

Comment:
Why provide a function that you then immediately call? Isn't it easier to not provide a function at all?
```suggestion
```

(and then drop all the indenting later on)

Reply:
Updated to remove the function.